tracer/src/Datadog.Trace/Propagators/SpanContextPropagator.cs

```diff
@@ -63,7 +63,7 @@ public static SpanContextPropagator Instance
 
             internal set
             {
-                if (value is null)
+                if (value == null!)
```

Comment:
You're not a fan of the `is` operator? 😱

Reply:
Haha, it's not that. I don't think I can silence the compiler warning with `!` if I use `is`.

Reply:
NULL!!!!